manifests/upstream/mongodb-operator/base/kustomization.yaml

```diff
@@ -0,0 +1,11 @@
+resources:
+  - https://raw.githubusercontent.com/percona/percona-server-mongodb-operator/v1.6.0/deploy/operator.yaml
+  - rbac.yaml
+
```

Comment:
```suggestion
namespace: mongodb-operator
```

Reply:
The mongodb-operator is pretty light-weight (just a single Deployment), do you want it to run in `platform-system` namespace instead of its own namespace?

Reply:
ok - lets do platform-system - but specify it here so that all the resources get correct namespace